prisma/migrations/20230814090235_add_webhook_as_relation/migration.sql

```diff
@@ -0,0 +1,21 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `webhooks` on the `User` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "User" DROP COLUMN "webhooks";
+
+-- CreateTable
+CREATE TABLE "Webhook" (
```

Comment:
hmm, I've noticed it a bit to late I guess - I think the table names should be in plural - so `Webhooks` instead of `Webhook`, `Users` instead of `User` and `Recipes` instead of `Recipe`